### PR TITLE
add initial raxxen check

### PR DESCRIPTION
**Pull request by rphillips**
#### Metric Sample
```
metric ovs_datapath.flow_count uint64 25
metric ovs_datapath.hit uint64 205189782
metric ovs_datapath.missed uint64 40026584
metric ovs_cpu double 0
metric ovs_datapath_masks.total uint64 11
metric ovs_datapath_masks.hit uint64 1614383088
metric ovs_datapath_masks.hit_per_pkt uint64 6.58
metric instance_count uint64 9
metric ovs_datapath.lost uint64 0
```

### CODE REVIEW

File: check/raxxen.lua [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+local fs = require('fs')
+local spawn = require('childprocess').spawn
+local LineEmitter = require('line-emitter').LineEmitter
+local misc = require('virgo/util/misc')
+
+local CONFIG_FILE = '/opt/rackspace/host.conf'
+local OVS_PID = '/var/run/openvswitch/ovs-vswitchd.pid'
+
+local function parseIni(filename)
+  local t = { ['DEFAULT'] = {} }
```
Comment (creationix) [inline check/raxxen.lua]: `DEFAULT` doesn't need quoting here.

Comment (rphillips) [inline check/raxxen.lua]: thanks. fixed

**Comment by creationix** [comment]
+1 I think this is good.  I'm not a fan of the OOP style, but I understand that how most agent code is written.

**Comment by stackedsax** [comment]
I mostly like 'raxxen' the word.  It sounds kinda exciting.

**Pull request merged**